Doc/whatsnew/3.13.rst

```diff
@@ -2130,6 +2130,9 @@ New Features
   between native integer types and Python :class:`int` objects.
   (Contributed by Steve Dower in :gh:`111140`.)
 
+* Add :c:func:`PyLong_GetSign` function to test sign of :class:`int` objects.
```

Comment:
Sadly, this PR is too late for Python 3.13, it missed the feature freeze (beta1). You should move this entry to Doc/whatsnew/3.14.rst.

Also:

```suggestion
* Add :c:func:`PyLong_GetSign` function to get the sign of :class:`int` objects.
```